packages/core/README.md

```diff
@@ -76,6 +76,14 @@ The mapConfiguration allows controlling many client instance details.
 | <...masterportalAPI.fields> | various          | The object is also used to initialize the masterportalAPI. Please refer to their documentation for options.                                                              |
 | <plugin.fields>             | various          | Many plugins added with `addPlugin` may respect additional configuration. Please see the respective plugin documentations. Global plugin parameters are described below. |
 | vuetify                     | object           | You may add vuetify configuration here.                                                                                                                                  |
+| extendedMasterportalapiMarkers | ExtendedMasterportalapiMarkers? | Optional. If set, all visible vector layers' features can be hovered and selected by mouseover and click respectively. They are available as features in the store. Please mind that this only works properly if you use nothing but point marker vector layers styled by the masterportalAPI. |
```

Comment:
Does this only work if I use only vector layers only including points, vector layers styled by the masterportalApi or vector layers only including points that are styled by the masterportalApi?  
Would this mean, that if I have this feature enabled and want to display lines or polygons, that weird or unexpected behaviour may happen?

Reply:
Yes, exactly. Lines, polygons, and differently styled points will, on hover, get the green marker style, too. I'm not even sure whether they'll appear as markers or briefly disappear and thus flicker on mouse movement.

https://github.com/Dataport/polar/pull/30/commits/0ed7edad39f341c76b39ed49526e2f45a038b8a0

Still yes, exactly. But now configurators whitelist which layers the effect is applied to. No idea why I tried to blacklist layers first, makes no sense.

Reply:
I like the new approach 🎩